### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by aldenpeterson-wf**
Resolves https://github.com/locustio/locust/issues/842

This also adds the automatic generation of the Changelog using [github_changelog_generator](https://github.com/github-changelog-generator/github-changelog-generator). This will mean a few things:

1. We will want to be more clear about having detailed PR descriptions as they are automatically added
2. We _could_ choose to not include closed issues without labels to have less clutter. I left (allthethings) here since this is the first changelog generation
3. In the future, releases will be much easier and can happen more frequently because this tool makes it trivial to generate release notes
4. We could replace the entire web changelog with the autogenerated one - I left it as a highlights and link to the full changelog. Thoughts?

When this is merged, someone with pypi access will have to publish the package (as well as tag; I'm not doing so until this is merged and someone else might beat me to it).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Release Process
+
+ * Run github_changelog_generator to update `CHANGELOG.md`
+ * update `locust/__init__.py` with new version number: `__version__ = "0.9.0.rc1"`
+ * set milestone to 0.9.0 on all included PR's
+ * update changelog in docs: `locust/docs/changelog.rst`
```
[the remark this answers is not in the document]
Comment (aldenpeterson-wf) [reply .github/CONTRIBUTING.md]: Well right now that changelog is still cherry picked - I could update it and say "this is the last release with a changelog manually curated" though and then leave a link to the autogenerated one.

**Comment by aldenpeterson-wf** [comment]
@cgoldberg I didn't add a Makefile target since right now it's still requiring a manual update step for the curated changelog.

If that ends up being archived, it'd be easier (could literally do `make release 0.9.0` type of syntax).

**Comment by aldenpeterson-wf** [comment]
@heyman @cgoldberg @mbeacom 

Anything we can do to speed this along? A ton of people have been asking about the next release and it'd be nice to get this released.

**Comment by anhpham1509** [comment]
Ping @heyman  and @cgoldberg if you have time to do the release this week?

**Comment by cgoldberg** [comment]
just released 0.9.0 to PyPI:
https://pypi.org/project/locustio/0.9.0/

:tada: 

done:

* updated Makefile
* landed this PR
* tagged 0.9.0
* built source and wheel distributions
* uploaded dists to PyPI

**Pull request merged**